docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -1,92 +1,94 @@
-import { Meta } from "@storybook/addon-docs";
-import { Figma } from "storybook-addon-designs/blocks";
+<Meta
+  title="Components/Selections/Combobox/Docs"
+  parameters={{ alpha: true }}
+/>
 
 # Combobox
 
-<Meta title="Components/Combobox/Combobox/Docs" />
+The `Combobox` is designed to provide a versatile and accessible interface for
+selecting options from a list. `Combobox` can be used for a variety of
+scenarios, such as selecting items from a predefined list, filtering and
+searching through data.
 
 ## Design & usage guidelines
 
-The Combobox will... _Add a brief description of the design and usage of the
-component_
+The `Combobox` component's primary function is to facilitate option selection
+and searching within a list of items. It should be used in scenarios where users
+need to choose from a set of options, with the added benefit of filtering and
+customization.
 
-<!--
-  What is the design purpose of this component? How do its responsibilities
-  relate to other components? What should this component not be used for?
-  Try to describe this component not in terms of what it looks like or what
-  elements make it up, but what function it performs or how it enables a user
-  to perform tasks (e.g. “Buttons allow users to initiate, complete, and reverse
-  actions.” vs “Buttons are rectangular elements that have a green background and white text”).
--->
+### Structure
 
-## Content guidelines
+`Combobox` is composed of three main parts:
 
-<!--
-  What type of content does this component present (documents, text, images, other components)?
-  What is the recommended, or expected, amount of content? What is the components’ behaviour when
-  the provided content diverges from what is expected (image aspect ratio, amount of text, video
-  vs audio, filesize)? Are there any important references to the [Product Vocabulary](https://atlantis.getjobber.com/guides/product-vocabulary)
-  that should be made regarding this component? What happens when content is unavailable or unreliable
-  (poor connection, empty states, flaky GPS data?)
--->
+#### Trigger
 
-## Related components
+The trigger is the user-facing interface to open and close the `Combobox` and
+initiate option selection. The choice between them depends on the design and
```

Comment:
What do you think about mentioning the 2 different trigger options that are currently available. I like the idea of making that information clear here. Thoughts?

Reply:
This is right below:

```
There are two primary options for trigger elements of the Combobox, TriggerButton and TriggerChip. You must have one trigger component.
```

I was planning on linking each web example - so where it says TriggerButton is a link to that web example.